src/plugins/saved_objects_management/public/services/actions/share_saved_objects_to_space_action.tsx

```diff
@@ -33,10 +33,10 @@ export class ShareToSpaceSavedObjectsManagementAction extends SavedObjectsManage
 
   public euiAction = {
     name: i18n.translate('savedObjectsManagement.shareToSpace.actionTitle', {
-      defaultMessage: 'Share to space',
+      defaultMessage: 'Assign spaces',
     }),
     description: i18n.translate('savedObjectsManagement.shareToSpace.actionDescription', {
-      defaultMessage: 'Share this saved object to one or more spaces',
+      defaultMessage: 'Change the spaces this object is assigned to',
     }),
```

Comment:
I mentioned above that I made this change so it would match the flyout title.
If we prefer "share" instead of "assign", we could instead change the flyout title, I just think all of the text should be consistent.

I prefer "assign spaces" / "assign object to spaces" because it still works if you're _removing the object from a space_. "Share" implies that you're only adding the object to other spaces.

Reply:
I'd really love to get someone from the @elastic/kibana-docs team to weigh in on the text here since I think it will be a pattern we'll want to follow as we grow this capability.

Reply:
Also, I feel like there's a possibility to call out future progress in this tooltip. Like "Coming soon" or something to indicate that we're in fact working towards making these objects shareable.
<img width="481" alt="Screen Shot 2022-03-28 at 19 09 52 PM" src="https://user-images.githubusercontent.com/549577/160502076-7a6b0187-cf31-40c2-8108-5fe20a5c73f4.png">

Reply:
If it's `all` spaces, shouldn't is use the Space avatar with the `*`?

Reply:
Second tooltip:
![Saved Objects second tooltip](https://user-images.githubusercontent.com/38990715/160629322-72416683-d878-43d6-b969-9ce817d43b3c.png)

What is the future plan for sharing saved objects? If we are going to start enabling users to share a space via the Share menu, then we should use `Share`, but if we are using a different configuration method, we should use the action the user is going to take. Also, let's move `Coming soon` to the title so that it's more obvious. For example:

`**Coming soon: [Assigned/Shared] saved object**`

`This saved object is available in only one space. In a future release, you can [assign/share] the saved object [to/in] multiple spaces.`


Third tooltip:
![Saved Objects third tooltip](https://user-images.githubusercontent.com/38990715/160635698-4c2037c9-fef8-49e1-a70b-826ee81bafba.png)

Can we make this a positive statement? For example:

`**Global saved object**`

`This saved object is available in all spaces.`

NOTE: I changed `object type` to `saved object` since I believe these are actually saved objects instead of types of saved objects. If I'm wrong, ignore my change.

Reply:
OK, changed in eec4ab7fd8007a7a3779cacd622e723e500c7fa1 --

First tooltip:
![image](https://user-images.githubusercontent.com/5295965/160661317-9fca997a-5288-43ee-8bea-45173ae6b7c5.png)

Second tooltip:
![image](https://user-images.githubusercontent.com/5295965/160660913-47e0bce8-303d-464b-9b8d-3e8c2561f6b9.png)

Third tooltip:
![image](https://user-images.githubusercontent.com/5295965/160661006-f65018b2-b221-4f42-a2e5-6fe4353cc609.png)

Reply:
🙌  Looks great. I would still use the avatar with `*` and add `isDisabled` because I think it's very important to discern between `this space only & not shareable` and `global & not shareable`.

Reply:
Third tooltip:
![image](https://user-images.githubusercontent.com/5295965/160665257-5740ed5e-ec21-44cc-be4e-65a326dfa312.png)

Reply:
Great! Sorry one last suggestion, can we append `and cannot be changed` to the Global object?

> This save object is available in all spaces and cannot be changed.

Reply:
Updated in 29fa30e1d1aff1b31873bbceaeaeda808aaca1f7 🎉 

![image](https://user-images.githubusercontent.com/5295965/160669524-8b0864e3-bc29-40bd-8248-b7c2a6842b8b.png)